fgpyo/util/metric.py

```diff
@@ -148,18 +148,25 @@ def _parsers(cls) -> Dict[type, Callable[[str], Any]]:
         return {}
 
     @classmethod
-    def read(cls, path: Path) -> Iterator[Any]:
+    def read(cls, path: Path, skip_extra: bool = True) -> Iterator[Any]:
```

Comment:
How about `ignore_extra_fields`?  In the docstring you also explain it as "ignore" which I think is a better term.